addons/stock/report/report_deliveryslip.xml

```diff
@@ -45,19 +45,19 @@
                     <h2>
                         <span t-field="o.name"/>
                     </h2>
-                    <div class="row mt32 mb32">
-                        <div t-if="o.origin" class="col-auto" name="div_origin">
+                    <div id="informations" class="row mt-4 mb-4">
```

Comment:
why do you need `id` here?

Reply:
I added it, to homogenize it with other reports. 

- https://github.com/odoo/odoo/blob/16.0/addons/sale/report/ir_actions_report_templates.xml#L43
- https://github.com/odoo/odoo/blob/16.0/addons/account/views/report_invoice.xml#L63